terragrunt/env/provider.tf

```diff
@@ -0,0 +1,20 @@
+# Generated by Terragrunt. Sig: nIlQXj57tbuaRZEa
```

Comment:
This file can be ignored and deleted - it'll get regenerated from your `terragrunt.hcl`

Reply:
added it to the .gitignore file